include/transport/span.h

```diff
@@ -17,277 +17,277 @@ using Span = std::span<T, N>;
 template<class T, std::size_t Extent = std::numeric_limits<std::size_t>::max()>
```

Comment:
This file is going to need an exception to the linter: The methods in this **MUST** resemble the **exact** methods of `std::span` otherwise you can no longer easily swap from C++17 to C++20 without breaking a lot. This file is awkward, and its existence is purely because of the C++17 requirement. Weird as it is, it needs to be exempt from the linter.